go.mod

```diff
@@ -3,32 +3,32 @@ module github.com/cheqd/did-resolver
 go 1.20
```

Comment:
Upstream go.mod is different. If changed, you might also need to change go versions in workflows and Dockerfile